### DISCUSSION TITLE
Custom Armor for Guards

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#8536] [Q&A] [answered]
Hello, I’m new to mine colonies.
Is it possible for guards to wear custom armor from other mods (like from epic knights mod) ?

I know there is some research to unlock higher Tiers of armor for them but I’d like to give them a custom armor from my mod, just for the looks.

Maybe it‘s possible to add a research which unlocks custom armor for them once u reach diamond.

**Comment by Raycoms** [comment, marked as answer]
Yes, they can wear custom armor if it is implemented the vanilla way

**Reply by ghost** [reply]
And what’s the vanilla way ? ^^
I also have them requesting mod items sometimes. For example my first guard is requesting 2 normal chainmail parts but also 2 modded items. 

Do the stats of the armor matter ? Or does it need to be like an “fake armor” like counting as a normal iron or diamond armor statwise just with different looks ?

Cause I tried to give them a modded armor sets which resembles iron armor statwise but he won’t put it on :/

**Reply by uecasm** [reply]
It just depends how the mod author implements it.  The stats themselves are irrelevant, it's what "tier" they assign it to.  (Usually they pick an appropriate tier for its stats, though not always.)

When you get the request, click on "more details".  It will flash through all the different acceptable armours, including both modded and vanilla.  Though don't forget that this only applies to unenchanted armour; armour with enchants raises its "level" and it might no longer be acceptable.
